pkgs/tools/misc/cinc-auditor/default.nix

```diff
@@ -0,0 +1,20 @@
+{ lib, ruby, bundlerApp, bundlerUpdateScript }:
+
+bundlerApp {
+  pname = "cinc-auditor-bin";
+  gemdir = ./.;
+
+  inherit ruby;
+
+  exes = ["cinc-auditor"];
+
+  passthru.updateScript = bundlerUpdateScript "cinc-auditor-bin";
+
+  meta = with lib; {
+    description = "Cinc Auditor is the community build of Inspec by Chef. Inspec is an open-source testing framework for infrastructure with a human- and machine-readable language for specifying compliance, security and policy requirements";
```

Comment:
description should not be that long. Anything lengthy should be written in longDescription

Reply:
I will change the description.